pkg/api/dashboard.go

```diff
@@ -77,6 +77,9 @@ func (hs *HTTPServer) GetDashboard(c *models.ReqContext) response.Response {
 	if rsp != nil {
 		return rsp
 	}
+
+	hasPublicDashboard, err := hs.DashboardService.HasActivePublicDashboard(c.Req.Context(), dash.Uid)
```

Comment:
should probably put this behind the feature toggle

Reply:
Yes, and maybe handle the `err` too.